src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2466,11 +2466,26 @@ public void ToggleBackgroundGridVisibility(object parameter)
             }
         }
 
+        /// <summary>
+        /// Updates grapic helpers (grid) inside the background preview VM
+        /// </summary>
+        /// <param name="parameter"></param>
+        public void UpdateGraphicHelpersScale(object parameter)
+        {
+            BackgroundPreviewViewModel.GridScale = PreferenceSettings.GridScaleFactor;
```

Comment:
could `BackgroundPreviewViewModel` be null? maybe `?`

Reply:
Yup, got it! Added the check.